src/server/game/DungeonFinding/LFGMgr.cpp

```diff
@@ -2150,13 +2160,31 @@ namespace lfg
         }
         else if (out && error == LFG_TELEPORTERROR_OK)
         {
+            // No dungon teleport allowed
+            if (!sWorld->getBoolConfig(CONFIG_LFG_TELEPORT))
+            {
+                std::ostringstream ss;
+                ss << "Please move to the Dungeon: " << dungeon->name;
+                ChatHandler(player->GetSession()).PSendSysMessage(ss.str());
+                return;
```

Comment:
no no no. we dont use hardcoded things in our core.

Reply:
Add it to acore_string

Reply:
void LFGMgr::TeleportPlayer(Player* player, bool out, WorldLocation const* teleportLocation /*= nullptr*/) 
This function is being called independent from line L#1729, in other parts of the code.